Product/Production/Adapters/General/CONNECTAdminGUI/src/main/java/gov/hhs/fha/nhinc/admingui/constant/NavigationConstant.java

```diff
@@ -38,6 +38,8 @@
     public static final String DIRECT_PAGE = "direct";
     public static final String DIRECT_XHTML = "direct.xhtml";
     public static final String CM_PAGE = "connectionManager";
+    public static final String PROPERTIES_PAGE="properties";
+    public static final String PATIENT_SEARCH_PAGE="patientDiscovery";
```

Comment:
Not needed for now, but I'd rename the xhtml.  patientdiscovery is only one part.  Could be universalClient.xhtml or we really should come up with a better name.